contracts/SampleRecipient.sol

```diff
@@ -9,6 +9,12 @@ contract SampleRecipient is RelayRecipient, Ownable {
 
     mapping (address => bool) public relaysWhitelist;
 
+    // Testing RelayHub: withdrawing a recipient's deposit is prohibited during relayed call
+    bool public withdrawDuringRelayedCall;
```

Comment:
There are no tests or test stubs for these variables yet, right?

Reply:
This var was added just to add tests to certain failure flows on RelayHub. Should probably refactor SampleRecipient and split it to several test recipients but that's for another PR.